docs/troubleshooting/authentication/my_app_is_starting_from_homepage_after_logging_in_once_in_the_run_mode.md

```diff
@@ -0,0 +1,25 @@
+---
+keywords: ['homepage', 'mode', 'after']
+slug: /my-app-is-starting-from-homepage-after-logging-in-once-in-the-run-mode
+title: My app is starting from HomePage after logging in once in the Run Mode
+---
+# My app is starting from HomePage after logging in once in the Run Mode
+
+If your app always starts from the HomePage after logging in once (especially in Run Mode), this is likely due to cached authentication or session data in your browser.
+
+## Troubleshooting Steps
+
+- **Clear your browser cache and history.**
+
+![](../../assets/20250430121300291232.png)
+
+- **Try a different browser** or use **incognito/private browsing mode** to see if the issue persists.
```

Comment:
Even in this case. Please check the doc contribution guidelines to see when to use bold. Please remove bold from all files here where its not following the guidelines.